sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -72,7 +72,7 @@
 @ServiceClient(builder = DataLakeFileSystemClientBuilder.class, isAsync = true)
 public class DataLakeFileSystemAsyncClient {
 
-    public static final String ROOT_FILESYSTEM_NAME = "$root";
+    static final String ROOT_FILESYSTEM_NAME = "$root";
```

Comment:
I think I mentioned in the api review that I'd rather keep this one public. It's not any sort of default value that we've chosen, and its less likely to change than our numeric constants, so I think it offers some good value for customers not having to hunt for or worry about this.

Reply:
will bring it back